src/main/java/com/gl/donate_receive/controller/ItemController.java

```diff
@@ -46,7 +50,7 @@ public String update(@PathVariable("itemId") String itemId, Model model) {
 	}
 
 	@PostMapping("/{itemId}")
-	public String update(@PathVariable("itemId") String itemId, Model model,
```

Comment:
If it's the update you're after - the put method should it be.
![star-wars-yoda-i11829](https://user-images.githubusercontent.com/108729712/177931376-712976c3-76d5-4693-b912-5a746ea3fd30.jpg)